### ISSUE TITLE
RA on Arch Linux not reading databases [after recent update]

### ISSUE THREAD

**Issue by ttofano** [libretro/RetroArch#10850]
Hey guys, hope you are all doing good!
So, I've looked all over the internet but couldn't find anything similar to my issue, don't know if it is a bug or not... (TL;DR version below)

I'm running RetroArch on `Arch Linux` and noticed, at first, that I couldn't add any new games to previously generated playlists. So I tried deleting all of my conf files (in `$HOME/.conf/retroarch`), reupdating databases and readding games to playlists. However, it never works. Nevertheless, I can play games just fine with `Load content` option from main menu, and I also checked CRCs and md5sums for the majority of my games, and they all check with No-Intro's databases.

Trying every solution I found on internet, I stumbled upon someone saying that my database files could be corrupt and that I should check within RA if databases were readable by accessing `Main Menu` > `Information` > `Database manager` > `Anydatabase.rdb`, but whenever I try this, RA freezes and I have to kill its PID.

I then realized that it started to happen after updating RA to 1.8.8 (from `pacman -Syu`). So I downloaded `1.8.5` binaries to some temp folder, ran it with my 1.8.8 config files and databases and everything works just fine! I can add games, download thumbnails, play games (with up-to-date cores, updated while running `RA 1.8.8`), check games' database entries, read databases (from `Information` menu), everything!

I have no idea what causes this issue (probably something about reading the database files in `1.8.8`?) or how to debug it, but I'll attach links to my files, if it helps...

### TL;DR version
My RA 1.8.8 cannot read database files, and because of that I cannot add new games to playlists... However, playlists are generated just fine in version 1.8.5 of RA using the very same conf and database files as in version 1.8.8. 

Here is what happens reading databases on each version:
- 1.8.5: https://pasteboard.co/JcYiWTn.png
- 1.8.8: https://pasteboard.co/JcYjqSC.png (causing RA to freeze so I have to `kill -9` its PID)

And when I try to add games from the `Scan directory` menu option:
- 1.8.5: https://pasteboard.co/JcYhSAF.png (as you can see on the bottom left corner, everything runs fine)
- 1.8.8: https://pasteboard.co/JcYiupN.png (progress bar stuck on bottom left corner; scanning never finishes)

### Environment information
- RetroArch: 1.8.8 (from Arch Linux's package manager)
-- Build date: May 27 2020
-- GIT version: 9552f87157
-- Compiler: GCC (10.1.0) 64-bit

- OS: Arch Linux 2020.06.01 (up-to-date)
-- Kernel: 5.7.2-arch1-1 x86_64 GNU/Linux

- RA conf: https://pastebin.com/LSXc1XhF
- RA latest run log (on version 1.8.8): https://pastebin.com/PkB4KhmR

Can someone help me, please? Thanks in advance!
Stay safe! ;)

**Comment by hizzlekizzle** [comment]
I believe this is a duplicate of https://github.com/libretro/RetroArch/issues/10711

**Comment by ttofano** [comment]
> I believe this is a duplicate of #10711

Hmm, yes it seems so. So it is a bug after all... Never occured to me that it could be a compilation problem, never looked for anything like it.

Well, I'll close this ticket, follow that one and wait patiently. I'm running a terrible flu, so too lazy to downgrade GCC, recompile, reinstall, yada-yada. I'll stick to 1.8.5 ftm hahaha... Sorry I can't contribute... Thanks, @hizzlekizzle ;)
